docs/spec/v1/kustomization.md

```diff
@@ -939,10 +949,13 @@ data:
   sops.vault-token: <BASE64>
 ```
 
-## Recommended settings
+## Working with Kustomizations
+
+### Recommended settings
```

Comment:
`Recommended settings` used to be inside this section, but we moved it out so that it could have a greater emphasis (ref: https://github.com/fluxcd/kustomize-controller/pull/822#discussion_r1152266697)